infrastructure/data-analysis/template.yaml

```diff
@@ -536,13 +536,6 @@ Resources:
       Protocol: 'email'
       TopicArn: !GetAtt SlackNotificationsTopic.TopicArn
 
-  EmailToSlackSNSTopicArn:
```

Comment:
can also remove SlackNotificationsSubscriptionGeneric and SlackNotificationsTopic as the actual slack channel txma-data-analysis-slack-notifications doesnt exist

Reply:
removed